deeplearning4j-nn/src/main/java/org/deeplearning4j/datasets/iterator/callbacks/DefaultCallback.java

```diff
@@ -0,0 +1,54 @@
+package org.deeplearning4j.datasets.iterator.callbacks;
+
+import org.nd4j.linalg.api.concurrency.AffinityManager;
+import org.nd4j.linalg.dataset.api.DataSet;
+import org.nd4j.linalg.dataset.api.MultiDataSet;
+import org.nd4j.linalg.factory.Nd4j;
+
+/**
+ * @author raver119@gmail.com
+ */
+public class DefaultCallback implements DataSetCallback {
```

Comment:
This one will be removed, it's for me only for perf testing, i wonder if it affects performance for any of existing RecordReaders.